Resources/Prototypes/Reagents/chemicals.yml

```diff
@@ -228,6 +228,22 @@
     - !type:AdjustWater
       amount: 1
 
+- type: reagent
+  id: Meth
+  name: meth
+  desc: Methamphetamine, more commonly know as meth, is a potent stimulant, with dangerous side-effects if too much is consumed.
+  physicalDesc: translucent
```

Comment:
nitpick but surely you can come up with a funnier physical description for methamphetamine than 'translucent'

Reply:
bruh, you tell me I dont' want "what does meth look like" on my recent searches.